mesonbuild/compilers/compilers.py

```diff
@@ -88,8 +88,10 @@
                   'vala': 'VALAFLAGS',
                   'rust': 'RUSTFLAGS'}
 
-# execinfo is a compiler lib on BSD
-unixy_compiler_internal_libs = ('m', 'c', 'pthread', 'dl', 'rt', 'execinfo')
+unixy_compiler_internal_libs = ('m', 'c', 'pthread', 'dl', 'rt')
+# execinfo is a compiler lib on FreeBSD and NetBSD
+if mesonlib.is_freebsd() or mesonlib.is_netbsd():
+  unixy_compiler_internal_libs += ('execinfo',)
```

Comment:
__[Flake8]__ indentation is not a multiple of four ([see on Sider](https://sider.review/gh/repos/19784232/pulls/6226#issue-4870259))

<details>

| Rule |
| - |
| E111 |

</details>
<sub>posted by Sider</sub>